assets/src/edit-story/elements/media/edit.js

```diff
@@ -120,6 +126,7 @@ function MediaEdit({ element, box }) {
     ref: setFullMedia,
     draggable: false,
     opacity: opacity / 100,
+    onClick: clearEditing,
```

Comment:
I tried just now and I see it currently already cancels editing when clicked on faded part. Is this needed?

Reply:
See comment on @miina 's review, this is now needed because I removed `pointer-events: none` in order to be able to capture the `wheel` event on the faded part

Reply:
👍

Reply:
@wassgha This might now conflict with https://github.com/google/web-stories-wp/issues/1192 though now -- based on [Sam's comment](https://github.com/google/web-stories-wp/issues/1192#issuecomment-631926009), the whole image area should be draggable (including the faded part). Thoughts?